dcicutils/ff_utils.py

```diff
@@ -387,6 +387,94 @@ def search_metadata(search, key=None, ff_env=None, page_limit=50, is_generator=F
         return search_res
 
 
+def get_item_facets(item_type, key=None, ff_env=None):
+    """
+    Gets facet information for the given item type from the given env
+    """
+    resp = get_metadata('/profiles/' + item_type + '.json', key=key, ff_env=ff_env)
+    facets = {}
+
+    # get direct facets
+    for query_url, info in resp.get('facets', {}).items():
+        facets[info['title']] = query_url
+
+    # status is hardcoded in search.py, so the same must be done here
+    facets['Status'] = 'status'
+
+    return facets
+
+
+def search_facets(**kwargs):
```

Comment:
It may be better form to break the typically-expected args out of the other kwargs. This could be: `def search_facets(key=None, ff_env=None, **kwargs)`